website/static/js/pages/render-nodes.js

```diff
@@ -2,9 +2,12 @@
 
 var $osf = require('js/osfHelpers');
 var $ = require('jquery');
+var ko = require('knockout');
 
 // model for components, due to simplicity did not create a new file
-var ComponentControl = {};
+var ComponentControl = {
+     anonymousUserName:ko.observable('<em>A user</em>')
```

Comment:
This doesn't need to be an observable, since it's value doesn't change during runtime.